fv/infrastructure/topology.go

```diff
@@ -166,6 +172,9 @@ func StartNNodeTopology(n int, opts TopologyOptions, infra DatastoreInfra) (feli
 		if opts.IPIPEnabled {
 			infra.SetExpectedIPIPTunnelAddr(felix, i, bool(n > 1))
 		}
+		if opts.VXLANEnabled {
+			infra.SetExpectedVXLANTunnelAddr(felix, i, bool(n > 1))
+		}
 
 		var w chan struct{}
 		if felix.ExpectedIPIPTunnelAddr != "" {
```

Comment:
Should probably wait for VXLAN IP to show up in the log too.  That removed a lot of flakes from the IPIP tests.